docs/services.md

```diff
@@ -1,114 +1,185 @@
+  * if your code is open-source and hosted on GitHub, use Quay to store container images
+  * if your code is private and hosted on GitLab, use Amazon ECR
 
-#### For Non-production Services
-The following categories are something we would expect a service to specify:
+For more information on Quay, see the [Quay][quay_docs].
 
-`Service Name` - The name of the service users or other services will attempt to consume i.e. `web-portal`
+For more information on ECR, see the [Amazon ECR documentation][ecr _docs].
 
-`Env` - The environment of the service i.e. `Dev`
 
-`Service` - The overall service name or project name i.e. `example-service`
 
-```
-<servicename>.<env>.<service>-notprod.homeoffice.gov.uk
+## Artifact Storage
 
-web-portal.dev.example-service-notprod.homeoffice.gov.uk
-```
+### Artifactory
 
-#### For Production Services
-As we want to protect production services from hitting limits and to create a distinction between services that are non-production, (not prod)  and production, we simplify the overall approach by using the main service name as the domain.
+ACP uses [Artifactory][artifactory] as its internal binary repository store for projects to push build time artifacts and dependencies to, such as jars and python modules.  
 
-`Service Name` - The name of the service users or other services will attempt to consume i.e. `web-portal`
+Before ACP began using Amazon ECR, we also stored Docker images on Artifactory. You can now access those images [here][docker_images]. Note that if an image is not downloaded for one year, we remove it from the repository.
 
-`Service` - The overall service name or project name i.e. `example-service`
+For more information on Artifactory, see the [JFrog Artifactory documentation][artifactory_docs].
 
-```
-<servicename>.<service>.homeoffice.gov.uk
 
-web-portal.example-service.homeoffice.gov.uk
-```
 
 ## Application Composition
-The following containers are typically used in applications hosted on ACP:
 
-#### Louketo Proxy (formerly Keycloak Proxy/Gatekeeper)
-[Louketo Proxy](https://github.com/louketo/louketo-proxy): a proxy that can be used to protect applications with Keycloak.
-#### Nginx Proxy
-[Nginx Proxy](https://github.com/UKHomeOffice/docker-nginx-proxy): for TLS and proxying your application container.
-#### Cert Manager
-[Cert-Manager](how-to-docs/cert-manager.md): for obtaining internal and external certificates.
+Applications hosted on ACP typically use the following containers:
+
+  * [Keycloak Gatekeeper][gatekeeper]: protects applications using [Keycloak][keycloak]. Note that in the future ACP is going to deprecate its usage.
```

Comment:
Deprecation is upstream (i.e ACP is no longer the maintainer of that project. The decision to deprecate was Red Hat's).